campaigns/eocb.md

```diff
@@ -0,0 +1,79 @@
+# The Education, Outreach & Community Building Campaign
+
+Education, Outreach and Community Building
+
+## Overview
+
+Success of the Pangeo project has lead to rapid improvement of Python-based
+and related tools and technology that can dramatically improve scientific
+data analysis.  Since Pangeo is an open community effort, its continued
+success depends upon the size and knowledge base of the community, itself.
+Hence, we must build and expand the community through education, outreach,
+and related community building activities.
+
+As more scientists are trained to use the Pangeo software stack, more potential
+contributors to that software stack will arise.  It is important to teach
+scientists who are new to Pangeo and the Pangeo stack to be comfortable being
+developers, themselves.  We need to be welcoming and supportive as a community,
+so that new "users" feel comfortable being "contributors," too.  We need to
+be able to teach not only how to use the software itself, but how to program
+using best practices, including git, GitHub, and how to contribute to the
+community through common communication channels such as GitHub, Discourse, and
+through regular open community meetings.
+
+## Vision
+
+As described in the [Xdev Vision Statements](../PURPOSE.md#xdev-vision-statements),
+the main goal of the EOCB Campaign is to build a community of more than just
+*users* of the Pangeo software stack and platform.  The goal is to build a
+community of "contributors" who feel welcome to contribute at any level they
+feel comfortable.  This involves many different areas of involvement.
+
+### Education in Software Usage
+
+The first step for many in their journey into Pangeo is learning how to
+*use* the software.  Ideally this will be accomplished with both online
+educational material including videos, text-based tutorials, live
+tutorials, and demonstration Notebooks.
+
+#### Possible Projects
+
+- Add to the Notebook example gallery (https://gallery.pangeo.io)
+- Create video tutorials and/or demos
+- Live tutorials
+- Online, self-paced educational material
```

Comment:
Online, self-paced interactive educational material (I thought we should make all materials having statistic tools to know how many people viewed and spent how long time, thus we can know the feedbacks from users automatically, and can change our material focus accordingly)

Reply:
This is an excellent suggestion, @halehawk!  I will add these ideas to the list.